### PR TITLE
Update README with common_folder instructions

### PR DESCRIPTION
**Pull request by haakonvt**

### CODE REVIEW

**Comment by haakonvt** [comment]
@ks93 Spellcheck & does-it-make-sense check please? 🙏

**Comment by haakonvt** [comment]
Example usage: https://github.com/cognitedata/deploy-functions/pull/11

**Review by pavelzubarev — APPROVED** [review]
LGTM, but I can judge the right use of `the`

**Comment by ks93** [comment]
Too late :(

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Deploy Cognite Function action
-This action deploys a Python function to Cognite Functions with a schedule defined.
+This action deploys a Python function to Cognite Functions, possibly with a schedule defined.
```
Comment (ks93) [inline README.md:2]:
```suggestion
This action deploys a Python function to Cognite Functions, optionally with a schedule defined.
```

File: README.md [hunk 2]
```diff
@@ -42,14 +43,36 @@ with:
...
     function_file: function1
-    function_folder: ./functions
+    function_folder: functions
+    common_folder: utilities
     function_secrets: ${{ secrets.COGNITE_FUNCTION_SECRETS }}
     schedule_file: schedule-${{ github.ref }}.yml
 ```
 
+### Common folder
+A common use case is that you do not want to replicate utility code between all function folders. In order to accomodate this, we copy all the contents in the folder specified by `common_folder` into the functions we upload to Cognite Functions. If this is not specified, we check if `common/` exists in the root folder and if so, _we use it_.
```
Comment (ks93) [inline README.md:53]:
```suggestion
Commonly, you do not want to replicate utility code between all function folders. In order to accomodate this, we copy all the contents in the folder specified by `common_folder` into the functions we upload to Cognite Functions. If this is not specified, we check if `common/` exists in the root folder and if so, _we use it_.
```

**Review by ks93 — COMMENTED** [review]
😮

**Pull request merged**